kinappserver/views_public.py

```diff
@@ -1435,3 +1435,39 @@ def get_discovery_apps():
     os_type = get_user_os_type(user_id)
 
     return jsonify(get_discovery_apps(os_type))
+
+
+
+@app.route('/contact-us', methods=['POST'])
+def contact_support_endpoint():
+    """create a new ticket in zendesk"""
+    import requests, json
+    payload = request.get_json()
+    print(payload)
```

Comment:
do we want to print the whole payload ?
think about what logs we want to see on production and use log instead of print